.github/scripts/notifications.py

```diff
@@ -0,0 +1,106 @@
+def extract_notifications(file_path):
+    notifications = {'app': {}, 'email': {}}
+    ignore_keys = ['buttonIntro', 'linkIntro', 'buttonLabel', 'footer', 'manageSettings', 'html']
+    with open(file_path, 'r', encoding='utf-8') as file:
+        for line in file:
+            line = line.strip()
+            if '=' in line:
```

Comment:
Should also check for the line not starting with `#` since `=` can occur in a comment.